nvme_gw_server.py

```diff
@@ -93,6 +93,26 @@ def bdev_rbd_create(self, request, context):
 
         return pb2.bdev_info(bdev_name=bdev_name)
 
+    def bdev_rbd_delete(self, request, context):
+        # Delete bdev
+        self.logger.info({
+            f"Received: {request.bdev_name}",
```

Comment:
nit: it would be nice to note that the request was to delete the bdev in the log